app/src/main/java/org/simple/clinic/registration/phone/PhoneNumberValidator.kt

```diff
@@ -28,43 +28,41 @@ class IndianPhoneNumberValidator : PhoneNumberValidator {
   override fun validate(number: String, type: PhoneNumberValidator.Type): Result {
```

Comment:
Can we fix the commit messages?

1. The tone is incorrect, it should be written in a way that it completes the sentence: "If applied, this commit will..."
2. We need to be more detailed in the commit messages. Instead of something like "Refactor `PhoneNumberValidator`", we should be saying exactly what is being done. In this case, it will be something like, "Extract validation logic into discrete methods in `PhoneNumberValidator`".